test/tests/transfers/multiapp_conservative_transfer/tests

```diff
@@ -0,0 +1,33 @@
+[Tests]
+  issues = '#12948'
+  design = '/MultiAppCopyTransfer.md'
+  [./conservative_transfer]
+    type = Exodiff
+    input = master_conservative_transfer.i
+    exodiff = 'master_conservative_transfer_out.e master_conservative_transfer_out_sub0.e'
+    requirement = 'MOOSE shall support conservative transfers'
+  [../]
+
+  [./conservative_transfer_subs]
+    type = Exodiff
+    input = master_power_density.i
+    exodiff = 'master_power_density_out.e master_power_density_out_sub0.e master_power_density_out_sub1.e'
+    requirement = 'MOOSE shall support conservative transfers'
```

Comment:
Requirements should be unique. You have two that are the same. They can be 90% the same, that's fine, what are you testing here and in the test above?